packages/horizontal-layout/src/vaadin-horizontal-layout-styles.js

```diff
@@ -27,4 +29,18 @@ export const horizontalLayoutStyles = css`
   :host([theme~='spacing']) {
     gap: 1em;
   }
+
+  ${enableLayoutImprovements
+    ? unsafeCSS`
+    ::slotted([style^='width: 100%']),
+    ::slotted([style*=' width: 100%']) {
```

Comment:
The `color: red;width: 100%` variant could also be covered:

```suggestion
    ::slotted([style^='width: 100%']),
    ::slotted([style*=' width: 100%']),
    ::slotted([style*=';width: 100%']) {
```

Reply:
This has become obsolete with using data attributes.